src/utils/getDev.ts

```diff
@@ -44,7 +44,7 @@ export const getDev = (device?: Device_3): GetDevResult => {
  * @see {@link ../../documents/device.md}
```

Comment:
**Suggestion:** The function `getDev` doesn't handle the case when `device` is undefined. This could lead to a runtime error if this parameter is not provided.
```suggestion
 export const getDev = (device?: Device_3): GetDevResult => {
     if (!device) {
         return { error: new Error('Missing required parameter: device') }
     }
     // ...
 }
```

Reply:
It is not true. The case is conver.